README.md

```diff
@@ -1,11 +1,28 @@
+<p align="center">
+<a href="http://kitura.io/">
```

Comment:
Apologies that this comment is also in the wrong place in the file - SQLite - it appears to be a sunset repo but we are linking to it in the list of plugins in here. Please can you check with Chris regarding whether we should remove this from the plugins list?